src/index.ts

```diff
@@ -245,11 +249,21 @@ async function deployService(serviceName: ServiceName, app: AppConfig) {
     ? ["deploy"]
     : ["versions", "upload", "--experimental-versions"];
 
-  return execa(
-    "npx",
-    ["wrangler", ...command, "--config", "./wrangler.toml", ...extraArgs],
-    options,
-  ).pipeStdout!(process.stdout);
+  const deployArgs = [
+    "wrangler",
+    ...command,
+    "--config",
+    "./wrangler.toml",
+    ...extraArgs,
+  ];
+
+  if (cronSchedule) {
+    extraArgs.push("--triggers", cronSchedule);
+  }
+
+  return execa("npx", [...deployArgs, ...extraArgs], options).pipeStdout!(
```

Comment:
extraArgs is added twice. I think you just need to place the `if (cronSchedule)` before `return execa`, and don't change what's in that return line